.github/workflows/deployTest.yml

```diff
@@ -61,16 +61,18 @@ jobs:
           node-version: "14"
       - name: Install dependencies
         run: yarn install
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: Update build variables
+        run: |
+          az config set extension.use_dynamic_install=yes_without_prompt
+          INSIGHTS_KEY=$(az monitor app-insights component  show -g prime-simple-report-test | jq -r '.[0].instrumentationKey')
+          echo "REACT_APP_APPINSIGHTS_KEY=${INSIGHTS_KEY}" > .env.production.local
```

Comment:
This gets me thinking maybe we should have a Makefile in here, as well? Though adding Makefiles alongside existing task-runners is weird, so probably I should make this a script in package.json if anything (maybe add to pre-build?)

Reply:
package.json sounds logical to me

Reply:
I decided to try extracting it to a custom action, and it is I think somewhat better on net (it's deployment-specific stuff, and it's a _really_ bad fit for package.json scripting). The composite-steps action type is clearly somewhat aspirational on github's part, since right now it's a verbose YAML way of writing a shell script, but I'm willing to get on board based on the theory that they'll keep making them better. 🤞